src/Artemis.Core/Models/Profile/Profile.cs

```diff
@@ -111,9 +124,17 @@ public override void Render(SKCanvas canvas, SKPointI basePosition, ProfileEleme
             foreach (ProfileScript profileScript in Scripts)
                 profileScript.OnProfileRendering(canvas, canvas.LocalClipBounds);
 
+            using var opacityPaint = new SKPaint();
+            if (Configuration.FadeInAndOut && Opacity < 1)
```

Comment:
Lets put everything inside this if statement to avoid unnecessarily using a layer when opacity is 1